lldb/source/Symbol/SwiftASTContext.cpp

```diff
@@ -1448,39 +1448,66 @@ void ApplyWorkingDir(SmallString &clang_argument, StringRef cur_working_dir) {
   llvm::sys::path::append(clang_argument, cur_working_dir, rel_path);
   llvm::sys::path::remove_dots(clang_argument);
 }
+
+std::array<StringRef> macro_flags = { "-D", "-U" };
+
+bool IsMultiArgClangFlag(StringRef arg) {
+  for (auto &flag : macro_flags)
+    if (flag == arg)
+      return true
```

Comment:
here's a syntax error

Reply:
I've pushed a fix, but github doesn't seem to recognize this fact :( Could you please restart CI checks after change becomes visible?

Reply:
Nice, the fix is here